vars/main.yml

```diff
@@ -23,6 +23,22 @@ core:
   upf:
     ip_prefix: "192.168.252.0/24"
     iface: "access"
+    # when using 'dpdk' mode please make sure atleast two VF devices are
+    # created out of 'data_iface'
+    mode: af_packet          # Options: af_packet or dpdk
+    # set to true if mode is dpdk. This will enable the required resources
+    # like hugepage, sriov, etc...
+    enable_resources: false
```

Comment:
Is not it possible to use `mode: dpdk` as an "umbrella" variable such that we do not have to create another variable (`enable_resources`)? For example, if `{{ core.upf.mode }} == dpdk`, then, enable resources, etc.

Reply:
done